code/modules/xenoarchaeology/traits/xenoartifact_malfunctions.dm

```diff
@@ -7,21 +7,22 @@
 	label_desc = "Parallel Bearspace Retrieval: A strange malfunction causes the Artifact to open a gateway to deep bearspace."
 	weight = 15
 	flags = URANIUM_TRAIT
-	var/bears //bear per bears
+	var/list/bears = list() //bear per bears
 
 /datum/xenoartifact_trait/malfunction/bear/activate(obj/item/xenoartifact/X)
-	if(bears < XENOA_MAX_BEARS)
-		bears++
-		var/mob/living/simple_animal/hostile/bear/new_bear = new(get_turf(X.loc))
-		new_bear.name = pick("Freddy", "Bearington", "Smokey", "Beorn", "Pooh", "Paddington", "Winnie", "Baloo", "Rupert", "Yogi", "Fozzie", "Boo") //Why not?
-		log_game("[X] spawned a (/mob/living/simple_animal/hostile/bear) at [world.time]. [X] located at [AREACOORD(X)]")
-	else
-		X.visible_message("<span class='danger'>The [X.name] shatters as bearspace collapses! Too many bears!</span>")
-		var/obj/effect/decal/cleanable/ash/A = new(get_turf(X))
-		A.color = X.material
-		qdel(X)
+	if(bears.len >= XENOA_MAX_BEARS)
+		return
+	var/mob/living/simple_animal/hostile/bear/new_bear = new(get_turf(X.loc))
```

Comment:
this is jank put get_turf onto its own var, also no need to get the loc of X since get_turf finds the nearest turf anyway